sweetest/src/main/java/com/mysugr/sweetest/framework/context/ConfigurationsTestContext.kt

```diff
@@ -2,7 +2,7 @@ package com.mysugr.sweetest.framework.context
 
 import com.mysugr.sweetest.framework.configuration.ModuleTestingConfiguration
 
-internal class ConfigurationsTestContext(private val factories: FactoriesTestContext) {
+class ConfigurationsTestContext(private val factories: FactoriesTestContext) {
```

Comment:
why `internal` removed here? as far as i can see you can still compile it when it's here.